package.json

```diff
@@ -59,13 +49,38 @@
     "mocha": "^8.1.3",
     "prettier": "^3.1.1",
     "tar": "^6.1.14",
-    "ts-node": "^10.9.1",
     "typescript": "^5.2.2"
   },
-  "dependencies": {
-    "bindings": "^1.5.0",
-    "node-addon-api": "^6.1.0",
-    "node-fetch": "^2.6.1",
-    "node-gyp": "^10.0.1"
-  }
+  "optionalDependencies": {
+    "@jazzer.js/core": "^2.1.0"
+  },
+  "main": "./dist/lib/index.js",
+  "types": "./dist/lib/index.d.ts",
+  "files": [
```

Comment:
we also specified this in the publish workflow:

```yml
- name: Create Github release with prebuilds
        if: steps.tag.outputs.tagcreated == 'yes'
        uses: softprops/action-gh-release@v2
        with:
          files: release/*
          tag_name: ${{ steps.tag.outputs.tagname }}
          fail_on_unmatched_files: true
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
do they serve different purposes? looks like `releases/*` only contains the binding files there

Reply:
Yes.  The releases are for the github release and the prebuilds get included in the npm bundle.  I added information explaining this in CONTRIBUTING.md